catalog/video/video-admin-plugin/src/main/java/org/codice/alliance/video/ui/service/StreamMonitorHelper.java

```diff
@@ -0,0 +1,209 @@
+import org.codice.alliance.video.stream.mpegts.UdpStreamMonitor;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.InvalidSyntaxException;
+import org.osgi.framework.ServiceReference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+
+public class StreamMonitorHelper implements StreamMonitorHelperMBean {
+
+    public static final String SERVICE_PID = "service.Pid";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StreamMonitorHelper.class);
+
+    private static final String MONITORING = "monitoring";
+
+    private static final String START_TIME = "startTime";
+
+    private ObjectName objectName;
+
+    private MBeanServer mBeanServer;
+
+    private BundleContext context;
+
+    public StreamMonitorHelper() {
+
+    }
+
+    @Override
+    public void callStartMonitoringStreamByServicePid(String servicePid) {
+        Map<String, StreamMonitor> udpStreamMonitors = getUdpStreamMonitorServices();
+
+        if (MapUtils.isEmpty(udpStreamMonitors)) {
+            return;
+        }
+
+        udpStreamMonitors.entrySet()
+                .stream()
+                .filter(entry -> entry.getKey()
+                        .equals(servicePid))
+                .forEach(stringStreamMonitorEntry -> {
+                    stringStreamMonitorEntry.getValue()
+                            .startMonitoring();
+                    LOGGER.debug("Started monitor for {}", stringStreamMonitorEntry.getKey());
+                });
+    }
+
+    @Override
+    public void callStopMonitoringStreamByServicePid(String servicePid) {
+        Map<String, StreamMonitor> udpStreamMonitors = getUdpStreamMonitorServices();
+
+        if (MapUtils.isEmpty(udpStreamMonitors)) {
+            return;
+        }
+
+        udpStreamMonitors.entrySet()
+                .stream()
+                .filter(entry -> entry.getKey()
+                        .equals(servicePid))
+                .forEach(stringStreamMonitorEntry -> {
+                    stringStreamMonitorEntry.getValue()
+                            .stopMonitoring();
+                    LOGGER.debug("Stopped monitor for {}", stringStreamMonitorEntry.getKey());
+                });
+    }
+
+    @Override
+    public List<Map<String, Object>> udpStreamMonitors() {
+        Map<String, StreamMonitor> udpStreamMonitors = getUdpStreamMonitorServices();
+
+        if (MapUtils.isEmpty(udpStreamMonitors)) {
+            return null;
+        }
+
+        return udpStreamMonitors.entrySet()
+                .stream()
+                .filter(entry -> entry.getValue() instanceof UdpStreamMonitor)
+                .map(stringStreamMonitorEntry -> {
+                    UdpStreamMonitor udpStreamMonitor =
+                            (UdpStreamMonitor) stringStreamMonitorEntry.getValue();
+                    Map<String, Object> map = new HashMap<>();
+                    map.put(Metacard.ID, stringStreamMonitorEntry.getKey());
```

Comment:
This ID isn't actually a metacard ID, right? If it isn't, then it's a bit confusing to use the `Metacard.ID` constant here.